### DISCUSSION TITLE
Tile card - Support showing additional information

### DISCUSSION THREAD

**Discussion by StijnVdd** [home-assistant/frontend#16121] [Accessibility]
I like the tile card really much but I feel it is missing some customizations.

It would be great if next to the state the tile card can also show another attribute of the entity.

Examples
- Contact sensor: show the last state change: in other words: how long ago was the door closed/ windows opened
- Binary sensor for garbage collection of plastics. Sensor is 'on' when collection is next day, 'off' otherwise. But the sensor has a attribute that shows the amount of days until the collection is scheduled.
- Switch: some switches have power measurement. This typically is a sensor and not a switch, but it would be great to show the power consumption of that device in the same tile card that toggles the switch

Can you please consider this addition?

**Comment by ciedema** [comment]
I'd be quite keen for this also.  The Mushroom Cards could do this via templates, which isn't the greatest solution, but at least it works.  It would be nice if the tile cards could display information from entities that belong to the same parent device.
